backend/new_task.php

```diff
@@ -0,0 +1,92 @@
+**************************************************************************
+    Program takes in argument from the front-end interface along with the corresponding account_id 
+    obtained from the db_connect file and stores the entries to the corresponding columns in the 
+    tasks table within the server
+    
+**************************************************************************
+
+<?php 
+
+session_start();
```

Comment:
What is the point of this? How come it is in new_task.php, but not new_account.php?